.github/workflows/supply-chain-security-validation.yaml

```diff
@@ -19,7 +19,7 @@ on:
         type: string
         required: false
       codeql-go-version:
-        default: "1.21"
+        default: "1.22"
```

Comment:
Better to remove it since after if statement this `{{ inputs.codeql-go-version || 'stable' }}` will be always I belive => default: "1.22"